plugins/source/oracledb/client/transformer.go

```diff
@@ -50,7 +188,9 @@ func GetValue(arr arrow.Array, i int) (any, error) {
 		if err != nil {
 			return nil, err
 		}
-		return toTime(a.Value(i)), nil
+		t := toTime(a.Value(i))
```

Comment:
This code was also used only in the tests during insert.
Apparently if you use Go `time.Time` type to insert data using the Oracle driver it allows only up to seconds precision. To fix this we need to use the built in `go_ora.TimeStamp`, see:

With `time.Time` we reach here:
https://github.com/sijms/go-ora/blob/41ca433ee475e64869448c27e6e20ad038613ae4/v2/parameter_encode.go#L93
https://github.com/sijms/go-ora/blob/41ca433ee475e64869448c27e6e20ad038613ae4/v2/converters/type_conversion.go#L1117

And with `go_ora.TimeStamp`:
https://github.com/sijms/go-ora/blob/41ca433ee475e64869448c27e6e20ad038613ae4/v2/parameter_encode.go#L116
https://github.com/sijms/go-ora/blob/41ca433ee475e64869448c27e6e20ad038613ae4/v2/parameter_encode.go#L116